README.md

```diff
@@ -16,3 +16,40 @@ You can see the full documentation of JSON-schema here.
 2. Create the example yaml data consulting the `schema.json` which is the result of converting original schema json - `missile-strike.schema.json`
 
 ![missilte-strike.yaml](./images/yaml.png)
+
+Specify schema in VScode.
+
+How to configure VS-Code to know which schema to use?
+
+First, install `YAML` extension
+![yaml extension](./images/yaml%20extension.png)
+
+Second, configure `YAML` extension settings
+![yaml extension settings](./images/yaml%20extension.png)
+
+And then go to settings.json
+
+![settings.json](./images/settings.png)
+
+In `settings.json` define `yaml.schemas`
+
+```json
+{
+  "workbench.colorTheme": "Default Dark Modern",
+  "editor.formatOnSave": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.tabSize": 2,
+  "git.confirmSync": false,
+  "git.enableSmartCommit": true,
+  "explorer.confirmDelete": false,
+  "javascript.updateImportsOnFileMove.enabled": "never",
+  "explorer.confirmDragAndDrop": false,
+  "redhat.telemetry.enabled": true,
+  "yaml.format.singleQuote": true,
```

Comment:
Thanks @Talented0217 .

Does this line specify the schema to use for this specific YAML file?

It does seem a cumbersome way of associating a schema with a YAML file.

Reply:
Aah, I see it can match a glob pattern.  So, if we have suffix (`xxx.missile.yaml`) then VS-Code will know which schema to use:
```
yaml.schemas: Helps you associate schemas with files in a glob pattern
```

Reply:
So all .missile.yaml file will take the same schema json?

Reply:
Yes - that seems like a good way of specifying which schema the file is associated with.  So we could have, `trial-12a.missile.yaml`,  `javelin.missile.yaml`, `hms-warrior.cannon.yaml`, `challenge-12a.tank.yaml`, etc, etc, etc.  It helps users to navigate their data-files _and_ it specifies which schema to use for each file.

Reply:
The best method for that, we can just define this line at the head of the yaml file.
like following code.
# yaml-language-server: $schema=schemas\schema.json

Reply:
I created the PR